compal/__init__.py

```diff
@@ -24,6 +24,14 @@
 LOGGER.setLevel(logging.INFO)
 
 
+class NatMode(Enum):
+    """
+    Values for NAT-Mode
+    """
+    on = 1
```

Comment:
`on` is not a python keyword pylint does not like it. I will rename this into `enabled`/`disabled` after the merge